relayer/processor/message_processor.go

```diff
@@ -28,6 +29,8 @@ type messageProcessor struct {
 	connMsgs      []connectionMessageToTrack
 	chanMsgs      []channelMessageToTrack
 	clientICQMsgs []clientICQMessageToTrack
+
+	localhost bool
```

Comment:
`isLocalhost` is slightly more readable, in that it clearly identifies the value as a bool. Otherwise, at first I assumed it was some kind of localhost object.